### PR TITLE
Content browser update

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| Versions      | master |

Reorganized the Content Browser section, added a mention of various views, including the new Tree view.

### CODE REVIEW

File: docs/content_model.md [hunk 1]
```diff
@@ -89,33 +89,39 @@ They represent three categories of Content Types that you can create
...
+
+To access the **Content Browser**, go to the **Content** tab and select **Content structure** or **Media**.
+Then, on the left side of the screen, click the **Browse** icon.
 
 ![Content browser in the menu](img/left_menu_browse.png "Content browser in the menu")
 
-User accounts, which are also considered content in eZ Platform, are viewable in the **Users** tab accessible in **Admin** (see [Users](site_organization/organizing_the_site.md#users)).
+With the **Content Browser**, you can switch between the Grid, Panels and Tree views to navigate through the content of the site.
+Depending on your permissions, you might be able to review the regular content, media, forms, site skeletons and User accounts.
+You can also use the **Content Browser** to search the Repository for content, select content options, create Bookmarks, and create new content when needed.
```
Comment (DominikaK) [inline docs/content_model.md:103]:
```suggestion
You can also use the **Content Browser** to search the Repository for content, select content options, create bookmarks, and create new content when needed.
```
"select content options" is unclear.

File: docs/content_model.md [hunk 2]
```diff
@@ -89,33 +89,39 @@ They represent three categories of Content Types that you can create
...
 
-During your work with eZ Platform you will often be asked to select content from the Repository.
-This happens for example when you want to move or copy a Content item, embed an image, link two Content items, etc.
-In such cases you will make use of the **Content Browser**.
+![Content Browser](img/udw.png "Content Browser")
```
Comment (DominikaK) [inline docs/content_model.md:105]: Remember to update the screenshot as well (may be in separate PR with all the other shots that will need to be updated)

File: docs/content_model.md [hunk 3]
```diff
@@ -89,33 +89,39 @@ They represent three categories of Content Types that you can create
...
 
-The **Content Browser** enables you to navigate through all content in the site, including regular content, media, and User accounts.
-You can also use it to search the repository for content, and to create new content when needed.
+!!! note
 
-![Content Browser](img/udw.png "Content Browser")
+    If you have administrator permissions, you can also view and manage user accounts and site skeletons in the **Admin** tab.
```
Comment (DominikaK) [inline docs/content_model.md:109]:
```suggestion
    If you have administrator permissions, you can also view and manage User accounts and site skeletons in the **Admin** tab.
```

File: docs/content_model.md (hunk 2 above)
Comment (dabrt) [reply docs/content_model.md:105]: The screenshot's been updated, but the change is so minute you might have missed it.

File: docs/content_model.md [hunk 4]
```diff
@@ -89,33 +89,39 @@ They represent three categories of Content Types that you can create
...
+In such cases you use the **Content Browser**.
+
+To access the **Content Browser**, go to the **Content** tab and select **Content structure** or **Media**.
+Then, on the left side of the screen, click the **Browse** icon.
 
 ![Content browser in the menu](img/left_menu_browse.png "Content browser in the menu")
 
-User accounts, which are also considered content in eZ Platform, are viewable in the **Users** tab accessible in **Admin** (see [Users](site_organization/organizing_the_site.md#users)).
+With the **Content Browser**, you can switch between the Grid, Panels and Tree views to navigate through the content of the site.
+Depending on your permissions, you might be able to review the regular content, media, forms, site skeletons and User accounts.
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/content_model.md:102]: Unfortunately, with the current implementation they are accessible wherever you visit the UDW

File: docs/content_model.md (hunk 1 above)
Comment (dabrt) [reply docs/content_model.md]: I am changing "select content options" to "edit content"
